chapters/01_part_getting_started/02_what_is_r.Rmd

```diff
@@ -4,7 +4,7 @@ At this point in the book, you should have installed R and RStudio on your compu
 
 ## What is data?
 
-Data is information about objects (e.g., people, places, schools) and observable phenomenon (e.g., weather, temperatures, and disease symptoms) that is recorded and stored somehow as a collection of symbols, numbers, and letters. So, data is just information that has be "written" down.
+Data is information about objects (e.g., people, places, schools) and observable phenomenon (e.g., weather, temperatures, and disease symptoms) that is recorded and stored somehow as a collection of symbols, numbers, and letters. So, data is just information that has to be "written" down.
```

Comment:
Hi @yuiiwase! Thank you so much for your edit. This is definitely a typo, but it should actually read, 'that has been "written" down.'